examples/azure-native-sample-project/README.md

```diff
@@ -0,0 +1,30 @@
+# Azure Native sample project
+
+This project creates a VM instance in Azure Cloud using the Pulumi Kotlin Azure native SDK.
+
```

Comment:
Could you also add a link to official Pulumi docs in each README.md file? 

IMO it should be immediately noticeable, so maybe we should make it **bold**?

Example:

**See [Azure Native: Installation & Configuration](https://www.pulumi.com/registry/packages/azure-native/installation-configuration/) page in Pulumi Registry for the official documentation.**